packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-stages.tsx

```diff
@@ -1,33 +1,46 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import type { ConnectedProps } from 'react-redux';
+import type { Dispatch } from 'redux';
 import {
   Pipeline,
   Stage,
   Description,
-  css,
   Link,
+  css,
+  spacing,
 } from '@mongodb-js/compass-components';
 
 import type { RootState } from '../../modules';
+import { stageAdded } from '../../modules/pipeline';
 
 const noStagesTextStyles = css({
   margin: 0,
 });
 
+const pipelineStyles = css({
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
 const PipelineStages: React.FunctionComponent<PipelineStagesProps> = ({
   stages,
+  onStageAdded,
 }) => {
   if (stages.length === 0) {
     return (
       <Description className={noStagesTextStyles}>
+        Your pipeline is currently empty. To get started select the{' '}
+        <Link onClick={() => onStageAdded()} hideExternalIcon>
+          first stage.
+        </Link>
```

Comment:
This should be a button, right?

Reply:
You can do `<Link as="button" ...` and apply a few additional styles to reset the styles to be normal link (we do this in a few other places). Visually it should be a link, yes, but semantically this is a button and we should try to follow these rules